admin/app/views/spree/admin/integrations/_form_actions.html.erb

```diff
@@ -0,0 +1,9 @@
+<% submit_text ||= integration.persisted? ? Spree.t(:update) : Spree.t('actions.connect') %>
```

Comment:
@mad-eel let's just use `new_resource_linkds `and `edit_resource_links` partials and not create new ones, we can live with `create` / `update` submit texts